src/iterated_integrals.jl

```diff
@@ -24,7 +24,7 @@ struct WikJGeneral_oop <: AbstractWikJGeneral
     m_seq::Matrix{Int}
     function WikJGeneral_oop(ΔW)
         m = length(ΔW)
-        M = m*(m-1)/2
+        M = Int(m*(m-1)/2)
```

Comment:
I'd suggest `M = div(m * (m - 1), 2)` instead. A quick check with
```julia
f(m) = div(m * (m - 1), 2); g(m) = Int(m * (m - 1) / 2)
```
shows that the output of `@code_native` and `@code_llvm` is a lot simpler for `f`.